fe/src/components/Statistics/StatisticsChart/StatisticsChart.re

```diff
@@ -51,4 +51,4 @@ let renderChart = (users, checkedUsersNids, data) => {
 let make = (~users, ~checkedUsersNids, ~chartData, _children) => {
   ...component,
   render: _ => renderChart(users, checkedUsersNids, chartData),
-};
```

Comment:
W tym pliku wyżej jest obliczanie stałej: usersToshow - wydaje mi się to niepotrzebe (tylko to co jest w if, to co w else jest dobre). Zaznaczanie 5 gdy nie ma żadnych jest już robione w reducerze gdy ładowane są dane userów do stanu.

Reply:
Kurde, nieźle wyłapałeś! Zmienione :).